Libraries/LibIPC/Forward.h

```diff
@@ -23,4 +24,22 @@ ErrorOr<void> encode(Encoder&, T const&);
 template<typename T>
 ErrorOr<T> decode(Decoder&);
 
+template<>
+ErrorOr<void> encode(Encoder&, Core::AnonymousBuffer const&);
+
+template<>
+ErrorOr<Core::AnonymousBuffer> decode(Decoder&);
+
+template<>
+ErrorOr<void> encode(Encoder&, Core::DateTime const&);
+
+template<>
+ErrorOr<Core::DateTime> decode(Decoder&);
+
+template<>
+ErrorOr<void> encode(Encoder&, Core::ProxyData const&);
+
+template<>
+ErrorOr<Core::ProxyData> decode(Decoder&);
+
```

Comment:
These don't belong here - the encoder declarations belong in Encoder.h, and the decoder delcarations in Decoder.h. We don't want every user of IPC/Forward.h to include Core/Forward.h.